### PR TITLE
feat(cli): wing it does not require a file name for single file folders

### LINKED ISSUE #1112: Default entrypoint for `wing it`
### Community Note

> Please vote by adding a 👍 reaction to the issue to help us prioritize.
> If you are interested to work on this issue, please leave a comment.

### Feature Spec

If you run `wing it` from a directory with a single `.w` file, there's no need to specify the name of the file.

Example:

```sh
$ mkdir bang
$ cd bang
$ echo "bring cloud;" > test.w
$ wing it
```

The file `test.w` is selected by default.

Because, why not?

### Use Cases

Ergonomics baby!

### Implementation Notes

_No response_

### Component

Other

### PR DESCRIPTION
**Pull request by attias**
This PR resolves #1112
command `wing it` with no file name will work if there's a single .w file in the directory

*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: apps/wing/src/commands/run.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      process.chdir(workdir);
+  
+      writeFileSync("foo.w", "bring cloud;");
+  
+      await expect(run("bar.w")).rejects.toThrow("bar.w doesn't exist");
+    
+    } finally {
+      process.chdir(prevdir);
+    }
+});
```
Comment (ShaiBer) [inline apps/wing/src/commands/run.test.ts:72]: Love the tests 💪

**Review by ShaiBer — COMMENTED** [review]
Congrats on your first PR @attias 🤗

**Comment by staycoolcall911** [comment]
@attias - note I have updated the PR title to reflect the new feature

**Review by staycoolcall911 — APPROVED** [review]
Nice work @attias - thank you for your contribution!
May I ask you to please move the test files under a new `test` folder (sibling to `src`)?

File: apps/wing/src/commands/run.test.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import * as open from "open";
```
Comment (staycoolcall911) [inline apps/wing/src/commands/run.test.ts:1]: Please move the test files under a new `test` folder (sibling to `src`).

**Pull request merged**